_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+
+No authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090"
+    }
+}]
+```
+
+Basic authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090",
+        "prometheus.auth.type" : "basicauth",
+        "prometheus.auth.username" : "admin",
+        "prometheus.auth.password" : "admin"
+    }
+}]
+```
+
+AWSSigV4 authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:8080",
+        "prometheus.auth.type" : "awssigv4",
+        "prometheus.auth.region" : "us-east-1",
+        "prometheus.auth.access_key" : "{{accessKey}}"
+        "prometheus.auth.secret_key" : "{{secretKey}}"
+    }
+}]
+```
+
+After configuring Prometheus in the `datasources.json` file, run the following command to load the configuration into the OpenSearch keystore. The configuration is securely stored as is contains credential information.
+
+```
+bin/opensearch-keystore add-file plugins.query.federation.datasources.config datasources.json
+```
+
+If you are updating the keystore during runtime, refresh the keystore using following API command:
+
+```bash
+POST /_nodes/reload_secure_settings
+{
+  "secure_settings_password":""
+  
+}
+```
+{% include copy-curl.html %}
+
+Metrics collected from Prometheus are displayed in the **Observability** application under **Metrics analytics**. For further information, see the [Prometheus Connector](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/prometheus_connector.rst) GitHub page.
+
+## Prometheus PPL queries
+
+Below is an example of a [Piped Processing Language]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index) (PPL) query using a Prometheus connection in OpenSearch. This example demonstrates a metric selecting query with specific dimensions:
+
+```
+source = my_prometheus.prometheus_http_requests_total | stats avg(@value) by span(@timestamp,15s), handler, code
+```
+
+## Metrics Analytics UI
+
+The Metrics Analytics UI is a new component in the **Observability** application for OpenSearch Dashboards, as seen in the following image, that allows you to create visualizations based on metrics collected by your OpenSearch cluster.
+
+![Metrics UI example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics1.png)
+
+You can save visualizations from created metrics in the **Metrics analytics** section of the **Observability** application as seen in the following image.
+
+![Metrics UI example 2]({{site.url}}{{site.baseurl}}/images/metrics/metrics2.png)
+
+Additionally, you can save a visualization created with [Event analytics]({{site.url}}{{site.baseurl}}/observing-your-data/event-analytics/) as seen in the following image.
```

Comment:
But if there are two ways to save you might want to create a procedure under creating with the instructions for how to save.

Reply:
Ok, I tried to clarify the difference here. Let me know what you think.